src/Printer.purs

```diff
@@ -2,6 +2,7 @@ module Printer (save) where
 
```

Comment:
@gaku-sei 
Perhaps the logics in this files could be broken down further, e.g into: parseCss, buildTemplate, renderTemplate 
🤔?

Reply:
Could be indeed. Normally the parse logic is in Parser (Utils for the bindings for the js `css` library). But we could split `build` and `render` for sure 😄